sdk/identity/Azure.Identity/src/PersistentTokenCache.cs

```diff
@@ -16,44 +16,45 @@ public class PersistentTokenCache : TokenCache
     {
         // we are creating the MsalCacheHelper with a random guid based clientId to work around issue https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/issues/98
         // This does not impact the functionality of the cacheHelper as the ClientId is only used to iterate accounts in the cache not for authentication purposes.
-        private static readonly string s_msalCacheClientId = Guid.NewGuid().ToString();
-
-        private static AsyncLockWithValue<MsalCacheHelper> cacheHelperLock = new AsyncLockWithValue<MsalCacheHelper>();
-        private static AsyncLockWithValue<MsalCacheHelper> s_ProtectedCacheHelperLock = new AsyncLockWithValue<MsalCacheHelper>();
-        private static AsyncLockWithValue<MsalCacheHelper> s_FallbackCacheHelperLock = new AsyncLockWithValue<MsalCacheHelper>();
-        private readonly bool _allowUnencryptedStorage;
-        private readonly string _name;
-
-        /// <summary>
-        /// Creates a new instance of <see cref="PersistentTokenCache"/>.
-        /// </summary>
-        /// <param name="allowUnencryptedStorage"></param>
-        public PersistentTokenCache(bool allowUnencryptedStorage = true)
-        {
-            _allowUnencryptedStorage = allowUnencryptedStorage;
-        }
+        internal static readonly string s_msalCacheClientId = Guid.NewGuid().ToString();
+        internal readonly bool _allowUnencryptedStorage;
+        internal readonly string _name;
```

Comment:
Are you making these internal for testing purposes? We have a bunch of other places we've done this in the Identity library but generally have always made them properties when making them internal rather than exposing the feilds.

Reply:
Yes, but in retrospect, the proper use of these fields is covered by other tests. I can revert them to private and delete some of the ctor validation test assertions.